scss/components/_navbar.scss

```diff
@@ -343,7 +343,7 @@
 
         .dropdown-toggle {
           position: absolute;
-          top: 0;
+          top: px(8);
```

Comment:
maybe these could be used without px() helper function?

Reply:
OK, that probably makes sense. How about if I review the use of px() in _navbar.scss (which is where it it used most heavily) and submit any changes as a separate PR?

Reply:
that would make most sense. as a matter of fact that's why I had that px() comment not tied to any one line of code